docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3'
+
+networks:
+  monitor-net:
+    driver: bridge
+
+services:
+
+  surveyor:
+    image: synadia/nats-surveyor
+    container_name:  nats-surveyor
+    volumes:
+      - $NATS_SURVEYOR_CREDS:/etc/surveyor/SYS.creds
+    entrypoint: /nats-surveyor
+    command: -c $NATS_SURVEYOR_SERVER_COUNT -creds /etc/surveyor/SYS.creds -s $NATS_SURVEYOR_SERVERS
+    ports:
+      - "7777:7777"
+    networks:
+      - monitor-net
+    labels:
+      org.label-schema.group: "nats-monitoring"
+
+  prometheus:
+    image: prom/prometheus:latest
+    container_name: prometheus
+    volumes:
+      - ./prometheus/:/etc/prometheus/
+      - $PROMETHEUS_STORAGE:/usr/local/share/prometheus
+    command: --config.file=/etc/prometheus/prometheus.yml --storage.tsdb.path=/usr/local/share/prometheus
+    ports:
+      - "9090:9090"
```

Comment:
maybe dont expose this by default as there is no auth, grafana will still be able to get to it without specifying these ports

Reply:
Good point, I'll remove that.